campaigns.json

```diff
@@ -1139,7 +1139,11 @@
     }, 
     "resize": "auto", 
     "secondaries": {
-      "/Neutrino_E-10_gun/RunIISummer17PrePremix-MCv2_correctPU_94X_mc2017_realistic_v9-v1/GEN-SIM-DIGI-RAW": {}
+      "/Neutrino_E-10_gun/RunIISummer17PrePremix-MCv2_correctPU_94X_mc2017_realistic_v9-v1/GEN-SIM-DIGI-RAW": {
+          "SecondaryLocation": [
+                    "T1_US_FNAL_Disk"
+                        ],
```

Comment:
There shouldn't be a comma here. I see the same issue in the following changes as well

Reply:
Yep I agree. I have now run jq . which checks the formatting. I hope that the SecondaryLocation placement is ok.